src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApprovedConsentOrderAboutToSubmitHandler.java

```diff
@@ -132,4 +133,23 @@ private Boolean isPensionDocumentsEmpty(Map<String, Object> caseData) {
         List<CaseDocument> pensionDocumentsData = documentHelper.getPensionDocumentsData(caseData);
         return pensionDocumentsData.isEmpty();
     }
+
+    private LocalDate getApprovalDate(Map<String, Object> caseData) {
+        return (caseData.get(CONTESTED_ORDER_DIRECTION_DATE) != null)
+            ? parseLocalDate(caseData, CONTESTED_ORDER_DIRECTION_DATE)
+            : parseLocalDate(caseData, CONSENTED_ORDER_DIRECTION_DATE);
+    }
+
+    static LocalDate parseLocalDate(Map<String, Object> caseData, String orderDirectionDateKey) {
```

Comment:
private not static.
Understanding why and when you would make a method static seems like a good area to spend some time on.